aten/src/ATen/native/mkldnn/xpu/detail/oneDNN.h

```diff
@@ -107,4 +107,30 @@ dnnl::memory::dims deconv_dst_size(
     IntArrayRef dst_padding,
     int64_t groups);
 
+at::Tensor quantized_convolution_pt2(
```

Comment:
What's the meaning of `_pt2` suffix?

Reply:
The `_pt2` formerly refers to quant op defined in pytorch 2.0. I removed such suffix due to we would only have quantized op defined in pt2e and there is no need to add this suffix for version distinction.